Unify the GKE and AKS cluster installation instructions (#5615)
kyma-project/kyma: docs/kyma/04-05-use-your-own-domain.md

- [Microsoft Azure](https://azure.microsoft.com)
- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.14.6 or higher
- Tiller 2.10.0 or higher
- [Docker](https://www.docker.com/)
- A [Docker Hub](https://hub.docker.com/) account
- [az](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)

>**NOTE:** Running Kyma on AKS requires three [`Standard_D4_v3` machines](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-general#dv3-series-1). You create these machines when you complete the **Prepare the cluster** step.

  </details>

</div>

## Choose the release to install

1. Go to [this](https://github.com/kyma-project/kyma/releases/) page and choose the release you want to install.

2. Export the release version as an environment variable. Run:

    ```bash
    export KYMA_VERSION={KYMA_RELEASE_VERSION}
    ```

## Set up the DNS

<div tabs name="dns-setup" group="use-your-own-domain">
  <details>
  <summary label="GKE">
  GKE
  </summary>

Delegate the management of your domain to Google Cloud DNS.

>**NOTE**: Google Cloud DNS needs to be set up only once per a DNS zone.

Follow these steps:

> 1. Export the project name, the domain name, and the DNS zone name as environment variables. Run thesecommands:

Reviewer: ```suggestion
1. Export the project name, the domain name, and the DNS zone name as environment variables. Run these commands:
```

    ```bash
    export GCP_PROJECT={YOUR_GCP_PROJECT}
    export DNS_NAME={YOUR_ZONE_DOMAIN}
    export DNS_ZONE={YOUR_DNS_ZONE}
    ```

2. Create a DNS-managed zone in your Google project. Run:

    ```bash
    gcloud dns --project=$GCP_PROJECT managed-zones create $DNS_ZONE --description= --dns-name=$DNS_NAME
    ```

>     Alternatively, create the DNS-managed zone through the GCP UI. In the **Network** section navigate to **Network Services**, click **Cloud DNS** and select **Create Zone**.

Reviewer: ```suggestion
    Alternatively, create the DNS-managed zone through the GCP UI. In the **Network** section navigate to **Network Services**, click **Cloud DNS**, and select **Create Zone**.
```

  <details>
  <summary label="GKE">
  GKE
  </summary>

- A domain for your [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) (GKE) cluster
- [Google Cloud Platform](https://console.cloud.google.com/) (GCP) project with Kubernetes Engine API enabled
- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.14.6 or higher
- [gcloud](https://cloud.google.com/sdk/gcloud/)
- [wget](https://www.gnu.org/software/wget/)

>**NOTE:** Running Kyma on GKE requires three [`n1-standard-4` machines](https://cloud.google.com/compute/docs/machine-types). You create these machines when you complete the **Prepare the cluster** step.

  </details>
  <details>
  <summary label="AKS">
  AKS
  </summary>

- A domain for your [Azure Kubernetes Service](https://azure.microsoft.com/services/kubernetes-service/) (AKS) cluster
- [Microsoft Azure](https://azure.microsoft.com)
- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.14.6 or higher
- Tiller 2.10.0 or higher
- [Docker](https://www.docker.com/)
- A [Docker Hub](https://hub.docker.com/) account
- [az](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)

> >**NOTE:** Running Kyma on AKS requires three [`Standard_D4_v3` machines](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-general#dv3-series-1). You create these machines when you complete the **Prepare the AKS cluster** step.

Reviewer 2: the same thing about **Prepare the AKS cluster** as above

but in general, I **really** like the new layout, pico bello 👍 

<img src="https://user-images.githubusercontent.com/17568639/65416951-3a868380-ddf9-11e9-81ea-ff225a92e74d.jpg" data-canonical-src="https://user-images.githubusercontent.com/17568639/65416951-3a868380-ddf9-11e9-81ea-ff225a92e74d.jpg" width="300" height="300" />

        -d "*.$SUB_DOMAIN.$DNS_DOMAIN"
    ```

    You will see the following message:

    ```bash
    Please deploy a DNS TXT record under the name
    _acme-challenge.rc2-test.kyma.online with the following value:

    # TXT_VALUE

    Before continuing, verify the record is deployed.
    ```

    Copy the `TXT_VALUE`.

4. Open a new terminal and export these environment variables:

    ```bash
    export DNS_DOMAIN={YOUR_DOMAIN} # example.com
    export SUB_DOMAIN={YOUR_SUBDOMAIN} # cluster (in this case the full name of your cluster is cluster.example.com)
    export RS_GROUP={YOUR_RESOURCE_GROUP_NAME}
    ```

5. Export the `TXT_VALUE`.

    ```bash
    export TXT_VALUE={YOUR_TXT_VALUE}
    ```

    To modify the TXT record for your domain, run:

    ```bash
    az network dns record-set txt delete -n "_acme-challenge.$SUB_DOMAIN" -g $RS_GROUP -z $DNS_DOMAIN --yes
    az network dns record-set txt create -n "_acme-challenge.$SUB_DOMAIN" -g $RS_GROUP -z $DNS_DOMAIN --ttl 60 > /dev/null
    az network dns record-set txt add-record -n "_acme-challenge.$SUB_DOMAIN" -g $RS_GROUP -z $DNS_DOMAIN --value $TXT_VALUE
    ```

> 6. Go back to the first console, wait about 2 minutes and press **Enter**.

Reviewer: ```suggestion
6. Go back to the first console, wait for about 2 minutes and press **Enter**.
```
